### DISCUSSION TITLE
Presentation

### DISCUSSION THREAD

**Discussion by hyanwong** [tskit-dev/what-is-an-arg-paper#190] [Show and tell]
For reference, here is a PDF of my probgen talk on this subject:

[ARG presentation.pdf](https://github.com/tskit-dev/what-is-an-arg-paper/files/8358340/ARG.presentation.pdf)

**Comment by JereKoskela** [comment]
Minor nitpick: you've missed out the second "k" in my surname on the final slide. Other than that, looks great!

**Reply by hyanwong** [reply]
Ah, sorry! Thanks for picking up on this. Not sure how I did that. Corrected now.
